tests/run_test.sh

```diff
@@ -114,27 +114,54 @@ if [[ "Q${INPUT_NEST02_NML:-}" != Q ]] ; then
     NPX_NEST=$NPX_NEST02; NPY_NEST=$NPY_NEST02
     K_SPLIT_NEST=$K_SPLIT_NEST02; N_SPLIT_NEST=$N_SPLIT_NEST02
     atparse < ${PATHRT}/parm/${INPUT_NEST02_NML} > input_nest02.nml
+else
+    lnbeg=$(grep -i -n "<output_grid_02>" model_configure | cut -d: -f1)
+    lnend=$(grep -i -n "</output_grid_02>" model_configure | cut -d: -f1)
+    [[ ! -z "$lnbeg" ]] && [[ ! -z "$lnend" ]] && sed -i -e "${lnbeg},${lnend}d" model_configure
 fi
 
 if [[ "Q${INPUT_NEST03_NML:-}" != Q ]] ; then
     INPES_NEST=$INPES_NEST03; JNPES_NEST=$JNPES_NEST03
     NPX_NEST=$NPX_NEST03; NPY_NEST=$NPY_NEST03
     K_SPLIT_NEST=$K_SPLIT_NEST03; N_SPLIT_NEST=$N_SPLIT_NEST03
     atparse < ${PATHRT}/parm/${INPUT_NEST03_NML} > input_nest03.nml
+else
+    lnbeg=$(grep -i -n "<output_grid_03>" model_configure | cut -d: -f1)
+    lnend=$(grep -i -n "</output_grid_03>" model_configure | cut -d: -f1)
+    [[ ! -z "$lnbeg" ]] && [[ ! -z "$lnend" ]] && sed -i -e "${lnbeg},${lnend}d" model_configure
 fi
 
 if [[ "Q${INPUT_NEST04_NML:-}" != Q ]] ; then
     INPES_NEST=$INPES_NEST04; JNPES_NEST=$JNPES_NEST04
     NPX_NEST=$NPX_NEST04; NPY_NEST=$NPY_NEST04
     K_SPLIT_NEST=$K_SPLIT_NEST04; N_SPLIT_NEST=$N_SPLIT_NEST04
     atparse < ${PATHRT}/parm/${INPUT_NEST04_NML} > input_nest04.nml
+else
+    lnbeg=$(grep -i -n "<output_grid_04>" model_configure | cut -d: -f1)
+    lnend=$(grep -i -n "</output_grid_04>" model_configure | cut -d: -f1)
+    [[ ! -z "$lnbeg" ]] && [[ ! -z "$lnend" ]] && sed -i -e "${lnbeg},${lnend}d" model_configure
```

Comment:
Can these three lines be replaced by something like:
`sed -i -e  "/<output_grid_04>/,/<\/output_grid_04>/d" model_configure`

Reply:
@DusanJovic-NOAA, I think your solution is simpler and better. Since the PR is already merged, please commit your change directly into your branch. Thanks a lot!